tests/lib/prepare-restore.sh

```diff
@@ -119,10 +120,12 @@ build_rpm() {
     mkdir -p "$rpm_dir/SOURCES"
     # shellcheck disable=SC2086
     (cd /tmp/pkg && tar "-c${archive_compression}f" "$rpm_dir/SOURCES/$archive_name" $extra_tar_args "snapd-$version")
-    if [[ "$SPREAD_SYSTEM" == amazon-linux-2-* || "$SPREAD_SYSTEM" == centos-* ]]; then
-        # need to build the vendor tree
-        (cd /tmp/pkg && tar "-cJf" "$rpm_dir/SOURCES/snapd_${version}.only-vendor.tar.xz" "snapd-$version/vendor")
-    fi
+    case "$SPREAD_SYSTEM" in
+        fedora-*|amazon-*|centos-*)
+            # need to build the vendor tree
+            (cd /tmp/pkg && tar "-cJf" "$rpm_dir/SOURCES/snapd_${version}.only-vendor.tar.xz" "snapd-$version/vendor")
```

Comment:
:heart:

Reply:
@mvo5 It has to exist for Fedora now because I don't conditionally request the source anymore. It always is requested, even if it isn't going to be used.